<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelTabTileWidget</class>
 <widget class="QWidget" name="LevelTabTileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_1">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeft</set>
       </property>
       <item>
        <widget class="QLabel" name="ampTypeLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Automap type :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ampTypeComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pillar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vertical wall</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Horizontal wall</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wall intersection</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vertical wall ending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Horizontal wall ending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Corner wall</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wall intersection (west)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wall intersection (east)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Horizontal wall (backside)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vertical wall (backside)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wall intersection (south)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="flagsGroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Automap flags</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Vertical:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="amp0">
          <property name="text">
           <string>Door</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="amp2">
          <property name="text">
           <string>Arch</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="amp4">
          <property name="text">
           <string>Grate</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Horizontal:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="amp1">
          <property name="text">
           <string>Door</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="amp3">
          <property name="text">
           <string>Arch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="amp5">
          <property name="text">
           <string>Grate</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="amp6">
          <property name="text">
           <string>Dirt</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="amp7">
          <property name="text">
           <string>Stairs</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeft</set>
       </property>
       <item>
        <widget class="QLabel" name="subtilesLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Subtiles :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="subtilesPrevButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Previous SubTile</string>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="subtilesComboBox">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="subtilesNextButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Next SubTile</string>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
